GraphQL-TSC.md

```diff
@@ -162,10 +162,6 @@ If you have questions about these processes, please contact [operations@graphql.
 
 ## Collaboration tools
 
-### Mailing List
-
-The GraphQL TSC can be reached at [tsc@lists.foundation.graphql.org](https://lists.foundation.graphql.org/g/tsc). The mailing list is open to anyone who has signed the GraphQL Specification membership agreement (for [individual](https://individual-spec-membership.graphql.org/) or [corporate](https://corporate-spec-membership.graphql.org/) members), but anybody can view the archives.
```

Comment:
It'd be nice to have a public way to contact the tsc. I'm fine to use GitHub for this if we don't want to manage email lists. See proposal below.